community/kernel/src/main/java/org/neo4j/kernel/impl/store/counts/keys/CountsKeyType.java

```diff
@@ -19,9 +19,11 @@
  */
 package org.neo4j.kernel.impl.store.counts.keys;
 
+import org.neo4j.kernel.impl.store.kvstore.UnknownKey;
+
 public enum CountsKeyType
 {
-    EMPTY( 0 ), ENTITY_NODE( 2 ), ENTITY_RELATIONSHIP( 3 ), INDEX_STATISTICS( 4 ), INDEX_SAMPLE( 5 );
```

Comment:
Let's postpone the removal of `EMPTY` since I guess  it is still use by the old code.